shoes-swt/lib/shoes/swt/list_box.rb

```diff
@@ -28,7 +28,7 @@ def initialize(dsl, parent)
       end
 
       def update_items
-        @real.items = @dsl.items.map(&:to_s)
+        @real.items = @dsl.items.to_a
```

Comment:
Looking back at this, I'm not sure if the `to_s` here is just a thoughtful guard or actually important. After the `to_a` I could add back the map. Opinions?